cdap-app-fabric/src/test/java/co/cask/cdap/gateway/handlers/AuthorizationHandlerTest.java

```diff
@@ -22,11 +22,14 @@
 import co.cask.cdap.common.UnauthorizedException;
 import co.cask.cdap.common.conf.CConfiguration;
 import co.cask.cdap.common.conf.Constants;
+import co.cask.cdap.common.http.CommonNettyHttpServiceBuilder;
+import co.cask.cdap.internal.app.services.http.AppFabricTestBase;
+import co.cask.cdap.proto.id.EntityId;
 import co.cask.cdap.proto.id.Ids;
 import co.cask.cdap.proto.id.NamespaceId;
 import co.cask.cdap.proto.security.Action;
-import co.cask.cdap.security.authorization.AuthorizationPlugin;
```

Comment:
There was you remove in your earlier addressed comments commit here: https://github.com/caskdata/cdap/commit/cb5be6a8e013933df05eb08f7306e8ccab3e75c0#diff-5aac67d8a7ee0e81bc041cf4bec4a75bL26

same for other